tests/test_routes.py

```diff
@@ -10,6 +10,73 @@
 #  UTF-8
 # ------------------------------------------------------------ #
 
+def test_shorthand_routes():
+    app = Sanic('test_shorhand_routes')
+
+    @app.get('')
```

Comment:
I couldn't get this to work with parameterization since it didn't like `@app.variable`.